src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.model;
+
+public class Fruit {
+    private String name;
+    private int quantity;
+
+    public Fruit(String name) {
+        this.name = name;
+        this.quantity = 0;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public void addQuantity(int quantity) {
+        this.quantity += quantity;
+    }
+
+    public void subtractQuantity(int quantity) {
+        this.quantity -= quantity;
+    }
```

Comment:
Handling the subtraction of quantity, there should be a check to ensure that the quantity does not become negative. If the resulting quantity after subtraction is negative, it would be appropriate to throw a RuntimeException to prevent invalid state.

Reply:
the logic was changed